src/LuaGlobals.cs

```diff
@@ -177,11 +183,19 @@ private void RegisterClassOrInterface(string path, Type type, int recursionCount
             {
                 if (
                     // Check that the LuaHideAttribute and LuaGlobalAttribute were not applied
-                    (!field.GetCustomAttributes(typeof(LuaHideAttribute), false).Any()) &&
-                    (!field.GetCustomAttributes(typeof(LuaGlobalAttribute), false).Any()))
+                    (!field.GetCustomAttributes(typeof(LuaHideAttribute), false).Any()))
                 {
+                    var name = field.Name;
```

Comment:
Avoid `var` when the type is not explicit or for primitive types

Reply:
Good catch! Fixed in several places :)